### PR TITLE
Edit search scroll docs for syntactic style

### PR DESCRIPTION
**Pull request by bobbyberg**

### CODE REVIEW

**Comment by clintongormley** [comment]
Correctness, or just style?  I'm not a java developer so i'm not sure which style is preferred.  I'll leave somebody else to decide on this but in the meantime, please could you sign the CLA so that it can be merged? http://www.elasticsearch.org/contributor-agreement/

File: docs/java-api/search.asciidoc [hunk 1]
```diff
@@ -60,17 +60,13 @@ SearchResponse scrollResp = client.prepareSearch(test)
...
     for (SearchHit hit : scrollResp.getHits().getHits()) {
         //Handle the hit...
     }
     scrollResp = client.prepareSearchScroll(scrollResp.getScrollId()).setScroll(new TimeValue(600000)).execute().actionGet();
-    //Break condition: No hits are returned
-    if (scrollResp.getHits().getHits().length == 0) {
-        break;
-    }
-}
+} while(scrollResp.getHits().getHits().length != 0);
```
Comment (bleskes) [inline docs/java-api/search.asciidoc]: can we add a comment again here like the on that was removed? something ala
```
// zero hits mark the end of the scroll and the end of the while loop.
```

**Comment by bleskes** [comment]
Agreed on that this is a style fix. That said - sure , no problem to pull it in. I left one minor comment. Once the CLA is signed (and the comment is applied), I'll pull it in.

**Comment by robertberg** [comment]
Thanks guys!

1. I've replaced the word "correctness" in the commit message with "style"
2. Signed the CLA
3. Added the commend proposed by Boaz

**Comment by bleskes** [comment]
Thx @robertberg

**Pull request merged**